ouroboros-consensus/src/Ouroboros/Consensus/Storage/ChainDB/Impl/Types.hs

```diff
@@ -642,6 +675,55 @@ deriving instance
   , InspectLedger blk
   ) => Show (TraceAddBlockEvent blk)
 
+-- | See 'DoneAddingBlock'
+data TraceDoneAddingBlockEvent blk =
+    TraceDoneAddingBlockEvent {
+        -- | The block itself
+        addedBlock             :: EventAuxData "addedBlock" blk
+        -- | The possibly-new ChainDB tip that resulted from this block having
+        -- been added
+      , addedBlockNewSelection :: Point blk
+        -- | The point of the added block
+      , addedBlockPoint        :: RealPoint blk
+
+        -- These fields are all times. They're ordered by expected value instead
+        -- of by name.
+
```

Comment:
I notice we use `Time`, `RelativeTime`, and `UTCTime`. Do we elaborate somewhere on why do we use different times?

Reply:
I added a sentence in the `TraceDoneAddingBlockEvent` data type's Haddock.